### PR TITLE
dkrutenko-3

### PR DESCRIPTION
**Pull request by dkrutenko**
added selenide, optimized softserve tests

### CODE REVIEW

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/rozetka/ShoppingCartPage.java [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-    }
+    private final String addToCartButtonSelector = "//*[@class='buy-button__label ng-star-inserted']";
+    private final String cartButtonSelector = "//rz-cart[@class='header-actions__component']";
+    private final String cartProductTitleLabelSelector = "//a[@class='cart-product__title']";
 
-    public void clickAddToCartButton() {
-        driver.findElement(addToCartButtonSelector).click();
+    public ShoppingCartModal clickAddToCartButton() {
+        $x(addToCartButtonSelector).click();
+        return new ShoppingCartModal();
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/rozetka/ShoppingCartPage.java:13]: add empty line before return

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/ContactUsPage.java [hunk 2]
```diff
@@ -1,51 +1,48 @@
...
+    private final String acceptTermsAndConditionsSelector = "//input[@name='isTermsAccepted']/ancestor::label";
+    private final String acceptUpdatesAndOffersSelector = "//input[@name='isUpdatedOffersAccepted']/ancestor::label";
 
-    public void fillContactPageFields(String firstNameText, String lastNameText, String emailText, String companyText,
-                                      String phoneNumberText, String messageText) {
+    public void fillContactPageFields(User user) {
         var firstName = $x(firstNameFieldSelector);
         firstName.click();
-        firstName.sendKeys(firstNameText);
+        firstName.sendKeys(user.getFirstname());
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/ContactUsPage.java:25]: you could create separate method for such operation in ElementsUtil and replace in places where such operations repeat
public void setText(String elementSelector, String text) {
var input = $x(elementSelector);
lastName.click();
lastName.sendKeys(text);
}

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/MainPage.java [hunk 3]
```diff
@@ -1,38 +1,34 @@
...
 
 public class MainPage {
 
-    private final By cookieButtonSelector = xpath("//button[@class='cookie-message__cta-button']");
-    private final By headerMenuContactUsButtonSelector = xpath("//button[@class='menu__contacts-button menu__contacts-button_black menu__contacts-button_black']");
-    private final By headerMenuButtonSelector = xpath("//button[@class='menu__button menu__button_background_white']");
-    private final By viewFullContactPageSelector = xpath("//a[@class='pop-up-form__external-link']");
-    private final By languageSwitcherSelector = xpath("//li[@class='language-switcher__item']");
-    private final By titleSelector = xpath("//p[@class='title__paragraph title__paragraph_listed smaller-font']");
+    private static final String cookieButtonSelector = "//button[@class='cookie-message__cta-button']";
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/MainPage.java:7]: non static, static only templates variables

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/User.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel;
+
+public class User {
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/User.java:3]: use lombok, it will replace constructor and getters

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel;
+
+public final class UserRepository {
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java:3]: UserRepo

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java [hunk 6]
```diff
@@ -0,0 +1,10 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel;
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java:1]: for repo shouls be package "repo"

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/User.java [hunk 7]
```diff
@@ -0,0 +1,63 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel;
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/User.java:1]: not usermodel, just models package

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+
+public final class UserRepository {
+
+    public User getContactUsUser() {
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java:5]: when you will start using lombok, here will be possible to use builder, make it static

File: src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/ElementsUtil.java [hunk 9]
```diff
@@ -1,4 +1,4 @@
-package com.softserveinc.ita.dkrutenko.utils.runners;
+package com.softserveinc.ita.mmakoviichuk.utils.runners;
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/ElementsUtil.java:1]: why did you move your class into mmakoviichuk?

File: src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/TestRunner.java [hunk 10]
```diff
@@ -11,9 +11,10 @@
 import java.util.concurrent.TimeUnit;
 
 public abstract class TestRunner {
+    public static final int defaultTimeout = 20;
 
     @Getter
-    private WebDriver driver;
+    protected static WebDriver driver;
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/TestRunner.java:17]: it's not your class, why did you made changes here?

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java [hunk 11]
```diff
@@ -2,69 +2,69 @@
...
+        contactUsPage.fillContactPageFields(user);
         contactUsPage.clickFormInputModalMenu();
-        contactUsPage.selectFormInputModalMenu(expectedCategory);
+        contactUsPage.selectFormInputModalMenu(user.getExpectedCategory());
 
         contactUsPage.clickAcceptTermsAndPolicyCheckbox();
         contactUsPage.clickAcceptUpdatesAndOffersCheckbox();
         var actualEmail = contactUsPage.getAttributeFromEmailField();
-        assertEquals(actualEmail, email);
+        assertThat(actualEmail.equals(user.getEmail()));
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:68]: separate all asserts from code before and use such usage of asserts (make this changes everywhere)

assertThat(actualText)
.as("Message if test fail: Text should be: " + expectedText)
.equals(expectedText);

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java [hunk 12]
```diff
@@ -2,69 +2,69 @@
...
+        softServeMainPage.clickHeaderMenuButton();
+        softServeMainPage.clickLanguageSwitcher();
         
-        var getTitle = softServeBasePage.getTitle();
+        var getTitle = softServeMainPage.getTitle();
         var expectedTitle = getTitle.contains(english)
                 || getTitle.contains(german)
                 || getTitle.contains(ukrainian);
-        assertTrue(expectedTitle);
+        assertThat(expectedTitle);
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:42]: assertJ should provide some method for such verifications
assertThat(expectedTitle)
.as("")
.contains(text1, text2, text3); <- take a look for such methods

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java [hunk 13]
```diff
@@ -2,69 +2,69 @@
...
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static com.codeborne.selenide.Selenide.open;
 
 public class SoftServeTests extends TestRunner {
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:15]: if you opening browsers in tests, not needed extending of testrunner

**Review by Nikodych — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/User.java (hunk 7 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/User.java:1]: done

File: src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/ElementsUtil.java (hunk 9 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/ElementsUtil.java:1]: i dont need my elemnts util class. When i pull from master - it was added into my lib

File: src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/TestRunner.java (hunk 10 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/TestRunner.java:17]: a was pushed master into my branch again. And there was some issues before demo. When i tried to start my tests - Makoviichuk's ElementsUtil class was sending error. Thats why i make it protected and static

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java (hunk 13 above)
Comment (dkrutenko) [reply src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:15]: but in testrunner i have my profile options and urls. Should i put them into test class?

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java (hunk 6 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java:1]: done

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java (hunk 5 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java:3]: done

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/MainPage.java (hunk 3 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/MainPage.java:7]: idk why i make it static lol

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/ContactUsPage.java (hunk 2 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/ContactUsPage.java:25]: done

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java (hunk 8 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java]: done

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java (hunk 12 above)
Comment (dkrutenko) [reply src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:41]: method .as doesn't work with contains

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java (hunk 11 above)
Comment (dkrutenko) [reply src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:67]: done

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/User.java (hunk 4 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/User.java]: done

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/rozetka/ShoppingCartPage.java (hunk 1 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/rozetka/ShoppingCartPage.java:14]: fixed

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java (hunk 12 above)
Comment (dkrutenko) [reply src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:41]: Method contains doesn't work with boolean expectedTitle method. If i put getTitle in assert, i can use contains but, i cant realize how to verify all three text's. If i use this ".contains(text1, text2, text3);" -> test will fail,  because other 2 text's doesn't available on the page

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java (hunk 13 above)
Comment (dkrutenko) [reply src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java]: Atm removed for softserve tests only

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java [hunk 14]
```diff
@@ -0,0 +1,40 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class User {
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java:7]: add @Builder here, and after this, you would be able to delete constructor and build the object in repo class in this way

User.builder().firtstName.build();

File: src/test/java/com/softserveinc/ita/dkrutenko/RozetkaTests.java [hunk 15]
```diff
@@ -38,16 +43,18 @@ public void verifyShoppingCartTest(String brandNameItem, String requiredItem) {
         //fill search with some brand name, click search button and click on required product
         searchGoods.fillSearchField(brandNameItem);
         searchGoods.clickSearchButton();
-        searchGoods.findActualtem(requiredItem);
+        ShoppingCartPage shoppingCartPage = searchGoods.findActualtem(requiredItem);
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/dkrutenko/RozetkaTests.java:46]: if are using var's, you should use it also for such variables var shoppingCartPage

File: src/test/java/com/softserveinc/ita/dkrutenko/RozetkaTests.java [hunk 16]
```diff
@@ -38,16 +43,18 @@ public void verifyShoppingCartTest(String brandNameItem, String requiredItem) {
...
         //add to cart, click continue and go on main page
-        shoppingCartPage.clickAddToCartButton();
+        ShoppingCartModal shoppingCartModal = shoppingCartPage.clickAddToCartButton();
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/dkrutenko/RozetkaTests.java:48]: same

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java [hunk 17]
```diff
@@ -2,69 +2,72 @@
...
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static com.codeborne.selenide.Selenide.open;
 
-public class SoftServeTests extends TestRunner {
-
-    private final MainPage softServeBasePage = new MainPage();
+public class SoftServeTests {
+    protected static final String softServeUrl = "https://www.softserveinc.com/";
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:15]: if it's used only in before method, move it directly there

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java [hunk 18]
```diff
@@ -2,69 +2,72 @@
...
-    public void verifyContactUsPage(String firstName, String lastName, String email,
-                                    String company, String phone, String message, String expectedCategory) {
-        softServeBasePage.clickAcceptCookieMessageButton();
-        softServeBasePage.clickHeaderContactsMenuButton();
-        ContactUsPage contactUsPage = softServeBasePage.clickViewFullContactPage();
+    @Test(dataProvider = "softServeContactUsFieldsVerification")
+    public void verifyContactUsFields(User user) {
+        softServeMainPage.clickAcceptCookieMessageButton();
+        softServeMainPage.clickHeaderContactsMenuButton();
+        ContactUsPage contactUsPage = softServeMainPage.clickViewFullContactPage();
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:60]: var

**Review by Nikodych — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/repo/UserRepo.java [hunk 19]
```diff
@@ -0,0 +1,12 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models.repo;
```
[the remark this answers is not in the document]
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/repo/UserRepo.java:1]: agreed

File: src/main/java/com/softserveinc/ita/dkrutenko/utils/runners/TestRunner.java [hunk 20]
```diff
@@ -1,63 +1,40 @@
...
+        driver = new ChromeDriver(options);
+        setWebDriver(driver);
+        open(rozetkaUrl);
     }
 
-    @AfterClass(alwaysRun = true)
-    public void tearDown() {
-        driver.manage().deleteAllCookies();
-        driver.quit();
+    @SneakyThrows
```
[the remark this answers is not in the document]
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/utils/runners/TestRunner.java:34]: there was Thread.sleep...

File: src/main/java/com/softserveinc/ita/dkrutenko/utils/runners/TestRunner.java [hunk 21]
```diff
@@ -1,63 +1,40 @@
...
-    protected LoginPageModal loginPageModal;
     protected static final String rozetkaUrl = "https://rozetka.com.ua/";
-    protected static final String softServeUrl = "https://www.softserveinc.com/";
+    private static final String chromeProfile = "C:/Users/IT/AppData/Local/Google/Chrome/User Data/";
     private static final String driverPath = "src/main/java/com/softserveinc/ita/dkrutenko/resources/chromedriver.exe";
-    @Getter
-    protected static WebDriver driver;
+    private WebDriver driver;
 
+    //uncomment "BeforeSuite" and "AfterMethod" for rozetka login test only
```
[the remark this answers is not in the document]
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/utils/runners/TestRunner.java:21]: Last test before commit was "rozetka login test"

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java (hunk 14 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java:7]: i dont know how to realize this...

Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java:7]: it should be after constructor ?

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java (hunk 17 above)
Comment (dkrutenko) [reply src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:15]: done

File: src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java (hunk 18 above)
Comment (dkrutenko) [reply src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java:60]: done

File: src/test/java/com/softserveinc/ita/dkrutenko/RozetkaTests.java (hunk 15 above)
Comment (dkrutenko) [reply src/test/java/com/softserveinc/ita/dkrutenko/RozetkaTests.java:46]: done

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java [hunk 22]
```diff
@@ -0,0 +1,40 @@
...
+        this.lastname = lastname;
+        this.email = email;
+        this.company = company;
+        this.phone = phone;
+        this.message = message;
+        this.expectedCategory = expectedCategory;
+    }
+
+    @Override
+    public String toString() {
```
[the remark this answers is not in the document]
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java:30]: yes, its working with "setText" method in ContactUsPage class

[the remark this answers is not in the document]
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java:30]: removed

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java (hunk 14 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java:8]: done

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/UserRepo.java [hunk 23]
```diff
@@ -0,0 +1,16 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models;
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/UserRepo.java:1]: create "repo" package -> com.softserveinc.ita.dkrutenko.pageobjects.softserve.repo;

**Review by Nikodych — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/UserRepo.java (hunk 23 above)
Comment (dkrutenko) [reply src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/UserRepo.java]: done

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/repo/User.java [hunk 24]
```diff
@@ -1,4 +1,4 @@
-package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models;
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.repo;
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/repo/User.java:1]: models/entities/enums in models package :)

**Review by Nikodych — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/entities/enums/UserRepo.java [hunk 25]
```diff
@@ -1,4 +1,4 @@
-package com.softserveinc.ita.dkrutenko.pageobjects.softserve.repo;
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models.entities.enums;
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/entities/enums/UserRepo.java:1]: repo classes should be in 'repo' package

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/entities/enums/User.java [hunk 26]
```diff
@@ -1,4 +1,4 @@
-package com.softserveinc.ita.dkrutenko.pageobjects.softserve.repo;
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models.entities.enums;
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/entities/enums/User.java:1]: I mean, that enum classes, model classes, and entity classes should be placed at 'models' package

User class is model class

**Review by Nikodych — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/repos/UserRepo.java [hunk 27]
```diff
@@ -1,4 +1,6 @@
-package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models.entities.enums;
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models.repos;
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/repos/UserRepo.java:1]: move out repos package from models package

**Review by Nikodych — CHANGES_REQUESTED** [review]

**Review by Nikodych — APPROVED** [review]
